.github/workflows/asan.yaml

```
@@ -12,11 +12,8 @@ jobs:
     name: rmf_task asan
     uses: open-rmf/rmf_ci_templates/.github/workflows/reusable_build.yaml@main
     with:
-      dist-matrix: |
-          [{"ros_distribution": "humble",
-            "ubuntu_distribution": "jammy"}]
       # NOTE: Avoid adding comments in the package lines, this can break some of the called scripts in github actions
       packages: |
         rmf_task
```

I guess we haven't updated these configs since we added `rmf_task_sequence` so we should append it to the list of packages after `rmf_task`

```suggestion
        rmf_task
        rmf_task_sequence
```

---

https://github.com/open-rmf/rmf_task/pull/90/commits/c038dc3dd18fa9d8f8963de1512a3f505d3f2a51